kernel-ewasm/validator/src/serialization.rs

```diff
@@ -1,27 +1,28 @@
-use crate::{io};
 use crate::primitives::*;
 pub use core::fmt;
 use pwasm_std::vec::{Vec};
 
 use pwasm_std::String;
 
+
 /// Deserialization from serial i/o.
-pub trait Deserialize : Sized {
+pub trait WASMDeserialize : Sized {
```

Comment:
Is this just to remove `cap9_core` as a dep?

Reply:
Yeah I wanted to make a clear distinction, because it's necessary for this trait to satisfy some requirements of the Instruction parser (which comes from parity). They are not the same requirements as syscalls, caps, etc.